doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-mortgages-example-proc.adoc

```diff
@@ -18,51 +18,67 @@ The package you select must contain all the required rule assets.
 . Click *Ok* to create and open the test scenario in the test scenarios designer.
 . Expand *Project Explorer* and verify the following:
 * `Applicant`, `Bankruptcy`, `IncomeSource`, and `LoanApplication` data objects exist.
-* `Pricing loans` guide decision table exists.
+* `Pricing loans` guided decision table exists.
 * Verify that the new test scenario is listed under *Test Scenario*
 . After verifying that everything is in place, return to the *Model* tab of the test scenarios designer and define the *GIVEN* and *EXPECT* data for the scenario, based on the available data objects.
 +
 .A blank test scenarios designer
-image:project-data/test-scenarios-preview-editor.png[]
+image:project-data/test-scenarios-blank-preview-editor.png[]
 +
 . Define the *GIVEN* column details,
 .. Click the cell named *INSTANCE 1* under the *GIVEN* column header.
 .. From *Test Tools* panel, select *LoanApplication* data object.
-.. Click *Add*.
-. To create properties of the data object, right-click the property cell and select *Insert column right* or *Insert column left* as required. For this example, you need to create two more property cells under the *GIVEN* column.
-. Click the first property cell,
+.. Click *Insert Data Object*.
+.. Optional: Click *Clear selection* to remove the selected data object.
+. To create properties of the data object, right-click the property header cell and select *Insert column right* or *Insert column left* as required. For this example, you need to create two more property cells under the *GIVEN* column.
+. Click the first property header cell,
 .. From *Test Tools* panel, select and expand the *LoanApplication* data object.
```

Comment:
Minor suggestion - add article:

s/From *Test Tools* panel,/From the *Test Tools* panel,